05/app.js

```diff
@@ -0,0 +1,56 @@
+const form = document.querySelector('form');
+form.noValidate = true;
+const messages = document.querySelector('.messages');
+const btn = document.querySelector('[type="submit"]');
+const required = form.querySelectorAll("[required]");
+const postCode = form.querySelector('[name="zip"]');
+let errors = [];
+
+form.addEventListener('submit', function(el){
+    el.preventDefault();
+
+    required.forEach(function(e){
```

Comment:
a Tutaj mamy `el` albo `item` - `e` kojarzy się z eventem, a to nie jest prawda.